pkg/disttask/framework/integrationtests/framework_test.go

```diff
@@ -234,22 +234,6 @@ func TestGC(t *testing.T) {
 	}, 10*time.Second, 500*time.Millisecond)
 }
 
-func TestFrameworkSubtaskFinishedCancel(t *testing.T) {
```

Comment:
this case test what we do when a subtask is cancelled by other routines by `CancelRunningSubtask`, it's actually duplicate with `TestFrameworkRunSubtaskCancelOrFailed`, so merge them